notebooks/05_gulf_stream_currents.ipynb

```diff
@@ -60,11 +1650,44 @@
     "\n",
     "m"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7b81a6df",
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62e2f498",
+   "metadata": {},
+   "source": [
+    "## Summary"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "248911ff",
+   "metadata": {},
+   "source": [
+    "In this example we loaded sea level data from an analysis-ready cloud based dataset and made a visualization of that data using mapping library."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0d17a736",
+   "metadata": {},
+   "source": [
+    "## Resources and references\n",
+    "- This notebook is based on the Pangeo physical oceanography gallery example: https://gallery.pangeo.io/repos/pangeo-gallery/physical-oceanography/05_gulf_stream_currents.html"
```

Comment:
This should turn the link into an actual link:
```suggestion
    "- This notebook is based on the Pangeo physical oceanography gallery example: <https://gallery.pangeo.io/repos/pangeo-gallery/physical-oceanography/05_gulf_stream_currents.html>"
```

Reply:
Good catch. I'll make the change.